operator/check_operator.go

```diff
@@ -191,4 +200,5 @@ type operatorCheck struct {
 	resolved                bool
 	policy                  policy.Policy
 	csvTimeout              time.Duration
+	subscriptionTimeout     time.Duration
```

Comment:
Do you need to default this in NewCheck?

Reply:
It gets defaulted in `root`

https://github.com/redhat-openshift-ecosystem/openshift-preflight/pull/1207/files#diff-44de5e8cb8a37326e7eab8c59d8fee1b9cd658555fcfe63af1da9d850cec8fd8R97

Reply:
Viper defaults it for the Preflight CLI, but the library caller isn't going through the same path, right? It's the same reason the [scorecard wait time](https://github.com/redhat-openshift-ecosystem/openshift-preflight/pull/1207/files#diff-6ff614f77f346e6e35530d89670adbcd1ef8b79971019a48350e7ad8065c353fR20) is preconfigured to a default for the library caller.

Reply:
I see what you mean now, updated the below, I'll need to go back and add this for CSV as well, in another PR.

https://github.com/redhat-openshift-ecosystem/openshift-preflight/pull/1207/files#diff-6ff614f77f346e6e35530d89670adbcd1ef8b79971019a48350e7ad8065c353fR29